docs/guides/cloud/data-storage-and-masking.mdx

```diff
@@ -0,0 +1,169 @@
+
+When [Test Replay](#Test-Replay) capturing is enabled, the additional data outlined below will be sent to the Cloud. You can disable capturing: TODO link this data.
+
+- The rendered DOM and CSS styles for the application under test
+- Cypress commands and events represented in the Command Log
+- Network traffic within your application under test
+- Browser console logs
+
+Cypress Cloud does not capture anything related to the code of your application under test or the code within any associated repositories that are associated with your project.
+
+## Cypress Cloud data cntrols
+
+We understand that controlling what data is sent to Cypress Cloud is important to our users. Cypress provides a number of ways to control what data is sent to Cypress Cloud.
+
+If you'd like more granular control over what data is sent to Cypress Cloud, please comment in this discussion: TODO:ADD A LINK TO DISCUSSION or [contact us](https://www.cypress.io/contact/) to discuss your needs.
+
+### Project access controls
+
+Cypress Cloud offers controls around the visibility of a project. You can set a project to be public or private. [See docs.](/guides/cloud/account-management/projects#Public-vs-Private)
+
+Setting a project to private means that users invited to the organization will have visibility into the project, its runs, and any data captured by Cypress.
+
+### Test Replay controls
+
+#### Disable Test Replay
+
+You can disable capturing data in Test Replay per project by following this guide: TODO: Test Replay guide link. This means Test Replay will not be available for viewing for the project. This will prevent Cypress Cloud from capturing the following data:
+
+- The rendered DOM and CSS styles for the application under test
+- Cypress commands and events represented in the Command Log
+- Network traffic within your application under test
+- Browser console logs
+
+### Screenshot controls
+
+#### Blackout elements in screenshots
+
+Cypress offers an option to black out (mask) an array of elements from screenshots. [See docs.](/api/commands/screenshot#Arguments) This allows you to black out specific elements on screenshot so that they will not be visible in the locally saved screenshots or to any users that have access to the run in Cypress Cloud.
+
+Add a control to your `cypress/support/index.js` file to set the default blackout options for all screenshots.
+
+```js
+before(() => {
+  Cypress.Screenshot.defaults({
+    // blackout all elements with 'mask-cy' attribute
+    blackout: ['mask-cy'],
+  })
+})
+```
+
+Add a control for a specific screenshot to blackout specific elements.
+
+```js
+it('admin page loads', () => {
+  cy.visit('/admin')
+  cy.screenshot({
+    blackout: ['.user-name'],
+  })
+})
+```
+
+#### Remove Cypress Command Log from screenshots
+
+Cypress offers an option to hide the Cypress Command Log from screenshots so that no information from that area will captured in screenshots. [See docs](/api/commands/screenshot#Arguments).
```

Comment:
same here for the link text & support path

Reply:
Addressed in [`afdd515` (#5425)](https://github.com/cypress-io/cypress-documentation/pull/5425/commits/afdd515eab51c8a1a5c402603b2ee5afd0640d69)